src/dotnet-install.ps1

```diff
@@ -215,6 +226,54 @@ function Get-CLIArchitecture-From-Architecture([string]$Architecture) {
     }
 }
 
+
+function Get-NormalizedQuality([string]$Quality) {
+    Say-Invocation $MyInvocation
+
+    if ([string]::IsNullOrEmpty($Quality)) {
+        return ""
+    }
+
+    switch ($Quality.ToLower()) {
+        { ($_ -eq "daily") -or ($_ -eq "signed") -or ($_ -eq "validated") -or ($_ -eq "preview")} { return $Quality.ToLower() }
+        #ga quality is available without specifying quality, so normalizing it to empty
+        { $_ -eq "ga" } { return "" }
+        default { throw "'$Quality' is not a supported value for --quality option, supported values are: daily, signed, validated, preview, ga. If you think this is a bug, report it at https://github.com/dotnet/install-scripts/issues." }
+    }
+}
+
+function Get-NormalizedChannel([string]$Channel) {
+    Say-Invocation $MyInvocation
+
+    if ([string]::IsNullOrEmpty($Channel)) {
+        return ""
+    }
+
+    switch ($Channel.ToLower()) {
```

Comment:
I would use if. Is the ToLower correct, the first value LTS is uppercase.

Reply:
We use these values to generate the download links which are case sensitive. LTS is upper case.